docs/usage/configuration-options.md

```diff
@@ -1862,21 +1862,21 @@ For example, if you have an `examples` directory and you want all updates to tho
 {
   "packageRules": [
     {
-      "matchPaths": ["examples/**"],
+      "matchFileNames": ["examples/**"],
```

Comment:
Line 1858 in this file says:

> Path rules are convenient to use if you wish to apply configuration rules to certain package files using patterns.

I think this line needs to be changed as well?

You should probably search for the string `path` in the `*.md` files, to make sure you've changed all the relevant docs.